### PR TITLE
[RNMobile] Enable edit button over Image blocks for Android

### PR DESCRIPTION
**Pull request by ashiagr**
## Description
This PR removes `showMediaEditorButton`  `const` references to enable edit button over Image blocks for Android as well.

The button was originally added as part of these PRs WordPress/gutenberg#19723, https://github.com/wordpress-mobile/gutenberg-mobile/pull/1782 and was enabled only for iOS till now.

`gutenberg-mobile` related PR:  https://github.com/wordpress-mobile/gutenberg-mobile/pull/2062
`WPAndroid` Issue: https://github.com/wordpress-mobile/WordPress-Android/issues/11147

(I don't have permissions on the main gutenberg repo, creating it in a fork-branch)

## How has this been tested?
Creating it as a draft PR as it still needs to be tested.

## Screenshots 
To be included once tested.

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [ ] My code is tested.
- [X] My code follows the WordPress code style. 
- [X] My code follows the accessibility standards. 
- [X] My code has proper inline documentation. 
- [X] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/image/edit.native.js [hunk 1]
```diff
@@ -607,8 +606,7 @@ export class ImageEdit extends React.Component {
 											! isUploadInProgress &&
 											! isUploadFailed &&
 											finalWidth &&
-											finalHeight &&
-											showMediaEditorButton && (
+											finalHeight && (
```
Comment (mchowning) [inline packages/block-library/src/image/edit.native.js]: You're deleting a `(` here, but I don't see that you're deleting the matching `)` anywhere?

Comment (ashiagr) [reply packages/block-library/src/image/edit.native.js]: I don't think so 🤔. Bracket was added to the previous line.

gutenberg_code

Comment (mchowning) [reply packages/block-library/src/image/edit.native.js]: You're right. Don't know what I was thinking. 🤦‍♂

**Review by mchowning — APPROVED** [review]
These changes are working well! 👍 

I've confirmed that the text alignment issue in the bottom sheet [that you observed](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2062#discussion_r398571762) is not due to any changes in this PR, so I think we can merge this.

**Review by etoledom — APPROVED** [review]
Agreed with @mchowning !
Tested on Android and WPAndroid via https://github.com/wordpress-mobile/WordPress-Android/pull/11557

Working like a charm! 🎉 

Thank you so much for this PR @ashiagr !

**Pull request merged**